File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    odd_add = 0
    even_add = 0
    for i in range(1, len(num_list)+1):
        if i % 2 != 0:
            odd_add += num_list[i-1]
        else:
            even_add += num_list[i-1]
    return max(odd_add, even_add)  

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    let odd_add = 0;
    let even_add = 0;
    for(let i=0; i<num_list.length; i+=2) {
        odd_add += num_list[i];
    }
    for(let i=1; i<num_list.length; i+=2) {
        even_add += num_list[i];
    }
    return Math.max(odd_add, even_add);
}